src/EPR.Payment.Service.Common.Data/Interfaces/IAppDbContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿using EPR.Payment.Service.Common.Data.DataModels.Lookups;
+using Microsoft.EntityFrameworkCore;
+
+namespace EPR.Payment.Service.Common.Data.Interfaces
```

Comment:
Replace the interface with the file I had sent earlier. The new file name is IFeesPaymentDataContext.cs

Reply:
resolved